src/components/App/App.test.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App.component';
+
+describe('Testing App component', () => {
+  let wrapper;
+  beforeEach(() => {
+    wrapper = render(<App />);
+  });
+
+  it('should to take snapshop', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+});
```

Comment:
good test, at least you will know that there is no crashing on render, but you will need to keep updating the snapshot on every change you made on the app structure/properties on components. 

You can add more tests oriented on validating the criteria acceptance, those kind of tests adds a lot of more value for the development perspective.